contribution/testing_guide.md

```diff
@@ -1,21 +1,9 @@
 # Testing Guide
 
-There are two ways to check the functionalities of KubeArmor: 1) testing KubeArmor manually and 2) using the testing framework.
+There are two ways to check the functionalities of KubeArmor: 1) testing KubeArmor manually and 2) using the testing framework. ALso if you want to test KubeArmor Controller or Operator then test by following [this](testing_operator_controller_guide.md) guide
+# 0. Make sure Kubernetes cluster is running 
 
-# 0. Make sure that the annotation controller is installed on the cluster (Applicable for Steps 1 and 2)
-
-- To install the controller from KubeArmor docker repository to your cluster run
-
-```text
-$ cd KubeArmor/pkg/KubeArmorAnnotation
-~/KubeArmor/pkg/KubeArmorAnnotation$ make deploy
-```
-- To install the controller (local version) to your cluster run
-
-```text
-$ cd KubeArmor/pkg/KubeArmorAnnotation
-~/KubeArmor/pkg/KubeArmorAnnotation$ make docker-build deploy
-```
+Although there are many ways to run a Kubernetes cluster (like minikube or kind) but it will not work with KubeArmor. If you will try to do this it will not identify your node. So use `k3s` or `kubeadm`. if you are new to these terms then the most easiest way to do this is by following this guide: [K3s installation guide](k3s/README.md)
```

Comment:
```suggestion
Although there are many ways to run a Kubernetes cluster (like minikube or kind) but it will not work with locally developed KubeArmor. KubeArmor needs to be on the same node as where the kubernetes nodes exist. If you will try to do this it will not identify your node since minikube and kind use virtualised node. You would either need to build your images and deploy them into these clusters or simply use `k3s` or `kubeadm`. if you are new to these terms then the most easiest way to do this is by following this guide: [K3s installation guide](k3s/README.md)
```